spec/datadog/profiling/integration_spec.rb

```diff
@@ -120,8 +120,9 @@ def stack_two
           @current_span = span
           @current_root_span = trace.send(:root_span)
           example.run
+
+          Datadog::Tracing.shutdown!
```

Comment:
Is tracing shutdown happening within a trace do .. end block a legit move? It feels odd.

Reply:
I've dropped a note to the tracing folks to ask them if this is OK or not. I think it can easily be reverted if needed, so I'll go ahead and merge the PR with this change.

Reply:
My concern is that this will cause side-effects in other tests (as it might mutate global state that isn't reset.)

My suggestions would be:

 - Add a mock to black hole the traces
 - Activate the "test" mode which introduces an adapter that intercepts outgoing payloads and puts them in an array instead.

Reply:
Opened https://github.com/DataDog/dd-trace-rb/pull/2456 to improve this.